orbs/helm/orb.yml

```diff
@@ -121,6 +121,37 @@ commands:
             git push origin $tag
 
 jobs:
+  build:
```

Comment:
The job should not be named `build` since it does not build anything. You should probably name it something such as `push` or `publish`.